tools/mason/MasonHelp.chpl

```diff
@@ -148,6 +148,7 @@ proc masonSearchHelp() {
   writeln();
   writeln("Options:");
   writeln("    -h, --help                  Display this message");
+  writeln("    --show                      Display the manifest of a single package's most recent version.");
```

Comment:
Since this is subject to change, I'd suggest keeping it more vague for now, e.g.

> Display the manifest of a package